src/main/java/network/brightspots/rcv/GuiConfigController.java

```diff
@@ -769,21 +770,22 @@ private boolean checkForSaveAndTabulate() {
     return willContinue;
   }
 
-  private void loadConfig(ContestConfig config) {
-    clearConfig();
+  // clear widgets then set to default values
+  // then populate widgets with values from input config
+  private void loadConfigIntoGui(ContestConfig config) {
+    clearWidgetsAndSetToDefaults();
     RawContestConfig rawConfig = config.getRawConfig();
 
-    OutputSettings outputSettings = rawConfig.outputSettings;
-    textFieldContestName.setText(outputSettings.contestName);
+    textFieldContestName.setText(config.getContestName());
     textFieldOutputDirectory.setText(config.getOutputDirectoryRaw());
-    if (outputSettings.contestDate != null && !outputSettings.contestDate.isEmpty()) {
+    if (!config.getContestDate().isEmpty()) {
```

Comment:
Make this same change for the other fields you changed to use `getString()`, i.e. getting rid of the != null check. I see other scattered instances throughout the code where this hasn't been done yet.